doc/src/user_guide.rst

```diff
@@ -238,23 +73,31 @@ The following commands are available from the ``pyfr`` program:
    found in the solution file.
 
 Running in Parallel
-^^^^^^^^^^^^^^^^^^^
+-------------------
 
 ``pyfr`` can be run in parallel. To do so prefix ``pyfr`` with
 ``mpiexec -n <cores/devices>``. Note that the mesh must be
 pre-partitioned, and the number of cores or devices must be equal to
 the number of partitions.
 
-Configuration File (.ini)
--------------------------
+.. _configuration-file:
 
-Overview
-^^^^^^^^
+Configuration File (.ini)
+=========================
 
 The .ini configuration file parameterises the simulation. It is written
 in the `INI <http://en.wikipedia.org/wiki/INI_file>`_ format.
 Parameters are grouped into sections. The roles of each section and
-their associated parameters are described below.
+their associated parameters are described below. Note that both ``;`` and 
+``#`` may be used as comment characters.
+
+Backends
+--------
+
+There are several configuration setting for the backends. The base 
```

Comment:
Rework English, is hard to follow, also correct spelling: "The base 
section ``[backend]`` is required, whereas PyFR has internal defualts for the 
other hardware specific backend settings; however, for best perfomance it is 
advisable to configure those applicable for the target hardware."